src/test/resources/rf2TestFiles/der2_ssccRefset_MRCMAttributeRangeSnapshot_INT_20170731.txt

```diff
@@ -89,3 +89,5 @@ fe14346d-fd26-49ab-a9db-ba74f8eca9ee	20170731	1	900000000000012004	723562003	246
 79cb1aae-7943-40e8-81ae-125379645f9f	20170731	1	900000000000012004	723562003	424361007	<< 105590001 |Substance (substance)|	<< 71388002 |Procedure (procedure)|: [0..*] { [0..1] 424361007 |Using substance| = << 105590001 |Substance (substance)| }	723597001	723596005
 7d23e837-4e2b-45cd-b5c8-8ea9b51daae0	20170731	1	900000000000012004	723562003	363702006	<< 404684003 |Clinical finding (finding)| OR << 71388002 |Procedure (procedure)|	<< 71388002 |Procedure (procedure)|: [0..*] { [0..1] 363702006 |Has focus| = (<< 404684003 |Clinical finding (finding)| OR << 71388002 |Procedure (procedure)|) }	723597001	723596005
 e6bbf042-3c52-4e16-be72-f780d002fb05	20210131	1	900000000000012004	723562003	3311482005	dec(#10..#20)	<< 373873005 |Pharmaceutical / biologic product (product)|: [0..*] { [0..1] (3311482005 |CD has presentation strength denominator value| >= #10 AND 3311482005 |CD has presentation strength denominator value| <= #20) }	723597001	723596005
+e6bbf042-3c52-4e16-be72-f780d002fb05	20210131	1	900000000000012004	723562003	3311482005	int(#30..#40)	<< 373873005 |Pharmaceutical / biologic product (product)|: [0..*] { [0..1] (3311482005 |CD has presentation strength denominator value| >= #10 AND 3311482005 |CD has presentation strength denominator value| <= #20) }	723597001	723596005
+e6bbf042-3c52-4e16-be72-f780d002fb05	20210131	1	900000000000012004	723562003	3311482005	str("test")	<< 373873005 |Pharmaceutical / biologic product (product)|: [0..*] { [0..1] (3311482005 |CD has presentation strength denominator value| >= #10 AND 3311482005 |CD has presentation strength denominator value| <= #20) }	723597001	723596005
```

Comment:
It would be good to use a different attribute id for the String example as it would be wrong for the same attribute to have 3 different range constraints.